apps/web/src/components/Alert/Alert.tsx

```diff
@@ -88,3 +92,56 @@ export const AlertProvider = ({ children }: { children: React.ReactNode }) => {
     </>
   );
 };
+
+type PropsV2 = {
+  children?: React.ReactNode;
+  className?: string;
+  id?: string;
+  variant?: "error" | "warning" | "information" | "success"; // Represents the color of the alert
```

Comment:
with twclassed, you don't have to manually define variants and potentially have it out of sync with `variantClasses`

alternatively in the future if you do it this way a better approach would be:
```
variant?: keyof typeof variantClasses
```